src/callbacks_step/analysis_dg1d.jl

```diff
@@ -13,8 +13,60 @@ function create_cache_analysis(analyzer,
 end
 
 
+function create_cache_analysis(analyzer, mesh::CurvedMesh,
+  equations::AbstractEquations{1}, dg::DG, cache,
+  RealT, uEltype)
+  # pre-allocate buffers
```

Comment:
We don't like this form of indentation either, don't know how this could happen, it's horrible. There is no difference between the following code and the function signature, there should be at least one indentation more to differentiate this.

We changed it to your suggestion to keep things consistently.